test/extended/csi/certification.go

```diff
@@ -0,0 +1,52 @@
+package csi
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	coreclient "k8s.io/client-go/kubernetes/typed/core/v1"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	configclient "github.com/openshift/client-go/config/clientset/versioned"
+)
+
+var _ = g.Describe("[sig-storage] [CSICertification] CSI driver", func() {
+	defer g.GinkgoRecover()
+
+	g.It("should be running on an OpenShift cluster", func() {
+		cfg, err := e2e.LoadConfig()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		c := configclient.NewForConfigOrDie(cfg)
+
+		coreclient, err := e2e.LoadClientset()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		g.By("checking for the Cluster Version Operator")
+		waitForCVO(coreclient.CoreV1().Namespaces())
+
+		cv, err := c.ConfigV1().ClusterVersions().Get(context.TODO(), "version", metav1.GetOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		clusterVersion, err := json.Marshal(cv)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		e2e.Logf("Detected cluster version: %s", clusterVersion)
```

Comment:
is this ever printed to openshift-tests stdout?

Reply:
It's not, removed